std/experimental/allocator/building_blocks/affix_allocator.d

```diff
@@ -168,10 +168,11 @@ struct AffixAllocator(Allocator, Prefix, Suffix = void)
         }
 
         static if (hasMember!(Allocator, "owns"))
-        Ternary owns(void[] b)
+        //Ternary owns(const void[] b) const ?
```

Comment:
Remove

Reply:
The question is, should the function header be
- `Ternary owns(const void[] b);` or
- `Ternary owns(const void[] b) const;`